contrib/systemd-with-workers/override-hardened-workers.conf

```diff
@@ -0,0 +1,71 @@
+[Service]
```

Comment:
Both `override-hardened-workers.conf` and `override-hardened.conf` appear to be the same files. This could cause issues when we decide to update these files (someone could accidentally updated one without copying the changes to the other).

I'd advise keeping just one of the files and instructing users to copy it to multiple places - unless we have good justification to have different security-related options specifically for workers. Which I'm not sure we do currently.

Reply:
This makes a lot of sense. I've made this change.